src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers/JavaCallableWrapperGenerator.cs

```diff
@@ -49,9 +51,13 @@ public OverriddenMethodDescriptor (string javaPackageName, string methodDescript
 			if (parts.Length > 2) {
 				Connector = parts[2];
 				if (parts.Length > 3) {
-					ManagedTypeName = parts[3];
+					ManagedTypeName = parts[3].Replace ('/', '+');
```

Comment:
Given that the reason for this isn't particularly clear, I think it would be helpful if we had a "nicely named method" "somewhere".

Question is what the method should be named, and where it should go.

Spending 2 seconds on it, I'd suggest putting it with `TypeDefinitionRocks`:

```csharp
partial class TypeDefinitionRocks {
    public static string CecilTypeNameToReflectionTypeName (string  typeName) => typeName.Replace ('/', '+');
}
```

Yes, it's longer, but at least it makes it clear *why* we're doing this.

Should probably update `TypeDefinitionRocks`' use of `.Replace('/', '+')` as well.